src/Microsoft.IdentityModel.Xml/XMLEncryptionConstants.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.IdentityModel.Xml
+{
+    /// <summary>
+    /// Constants for XML Encryption
+    /// Definitions for namespace, attributes and elements as defined in http://www.w3.org/TR/xmlenc-core1/
+    /// </summary>
+    public static class XmlEncryptionConstants
+    {
+#pragma warning disable 1591
+        public const string Namespace = "http://www.w3.org/2001/04/xmlenc#";
+
+        public const string Prefix = "xenc";
+
+        public static class Attributes
```

Comment:
does this library use embedding of types inside of other types? If not I would have suggested moving to separate files and separate by namespace instead of using embedded types, thoughts?

Reply:
This is more of a utility class. While coding it's easy to find a specific constant i.e. an Attribute by typing XmlEncCostants.Attribute. 

Not really sure if I understand - Are you suggesting that it would be a good idea to create a class hierarchy based on an xenc xsd schema e.g. EncryptionMethod contains only _Algorithm_ as an attribute and _KeySize_, _OAEPparams_ as elements.

Reply:
addressed with: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/commit/3ebd2b74bc0cae863ea82afcba6448e64021dc2e